norfair/drawing.py

```diff
@@ -360,6 +363,8 @@ def draw_tracked_boxes(
                 )
 
         if id_size > 0:
+            points = obj.estimate
+            points = points.astype(int)
```

Comment:
Thanks for catching this bug! I would create the `points` variable before the `if draw_box:` line, so that it is not computed twice in case `draw_box` is set to `True`